Makefile

```diff
@@ -351,3 +351,39 @@ install: _output/local/bin
           done ; \
         fi
 endif
+
+define RELEASE_HELP_INFO
+# release components.
+#
+# Args:
+#   WHAT: Component names to be release. support: kubeedge/edgesite/keadm
+#         If not specified, "everything" will be build and release.
+#
+# Example:
+#   make release
+#   make release HELP=y
+#   make release WHAT=kubeedge/edgesite/keadm
+#   make release WHAT=kubeedge GOARM=GOARM7/GOARM8
+#
+endef
+.PHONY: release
+ifeq ($(HELP),y)
+release:
+	@echo "$$RELEASE_HELP_INFO"
+else ifeq ($(WHAT),edgesite)
+release:
+	KUBEEDGE_OUTPUT_SUBPATH=$(OUT_DIR) hack/make-rules/build.sh edgesite-server edgesite-agent
+	hack/make-rules/release.sh "edgesite" $(GOARM)
+else ifeq ($(WHAT),keadm)
+release:
+	KUBEEDGE_OUTPUT_SUBPATH=$(OUT_DIR) hack/make-rules/build.sh keadm
+	hack/make-rules/release.sh "keadm" $(GOARM)
+else ifeq ($(WHAT),kubeedge)
+release:
+	KUBEEDGE_OUTPUT_SUBPATH=$(OUT_DIR) hack/make-rules/build.sh cloudcore admission edgecore csidriver iptablesmanager
+	hack/make-rules/release.sh "kubeedge" $(GOARM)
+else
+release:
+	KUBEEDGE_OUTPUT_SUBPATH=$(OUT_DIR) hack/make-rules/build.sh
+	hack/make-rules/release.sh "kubeedge" "edgesite" "keadm" $(GOARM)
+endif
```

Comment:
It's better to move the if else logic to the release script :)

Reply:
fixed.